src/components/CalendarDay.jsx

```diff
@@ -46,6 +48,16 @@ export default class CalendarDay extends React.Component {
     onDayMouseLeave(day, e);
   }
 
+  renderDay(day) {
+    const { renderDay } = this.props;
+
+    if (renderDay) {
+      const result = renderDay(day);
+      return typeof result === 'string' ? result : Children.only(result);
```

Comment:
Okay, after more exploration, we kind of realized that we should probably just return the result directly because `Children.only` breaks on `null`, `string`, `number`, `undefined`... etc. X_x I thought this function did something slightly different then it does.

Basically we should just have {renderDay ? renderDay(day) : day.format('D')}` directly in the `<td />` instead.